modules/kubernetes-addons/aws-cloudwatch-metrics/locals.tf

```diff
@@ -0,0 +1,48 @@
+locals {
+  name                 = "aws-cloudwatch-metrics"
+  namespace            = "amazon-cloudwatch"
+  service_account_name = "cloudwatch-agent"
+
+  set_values = [
```

Comment:
most locals have this prop towards the bottom. We should try and keep ordering consistent.

Reply:
addressed